app/CineHub/app/src/main/java/org/cinehub/HomeActivity.java

```diff
@@ -0,0 +1,46 @@
+package org.cinehub;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import androidx.fragment.app.FragmentTransaction;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import org.cinehub.api.CinehubAPI;
+import org.cinehub.api.model.SeatReservation;
+import org.cinehub.api.model.User;
+
+public class HomeActivity extends NavActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+
+        TextView tvHomeTitle = findViewById(R.id.tvHomeTitle);
+        TextView tvHomeSub = findViewById(R.id.tvHomeSub);
+        FloatingActionButton fabAddTicket = findViewById(R.id.fabAddTicket);
+
+        User user = getIntent().getParcelableExtra(LoginActivity.EXTRA_USER);
+        CinehubAPI api = new CinehubAPI();
```

Comment:
This is not the way to interact with the API. The use of the interfaces should be the only way to interact with it